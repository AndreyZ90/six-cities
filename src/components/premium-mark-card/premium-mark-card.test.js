import React from 'react';
import TestRenderer from 'react-test-renderer';

import PremiumMarkCard from '@/components/premium-mark-card/premium-mark-card';

describe(`PremiumMarkCard component snapshot`, () => {
  test(`Should correctly render PremiumMarkCard component`, () => {
    const tree = TestRenderer.create(<PremiumMarkCard />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
