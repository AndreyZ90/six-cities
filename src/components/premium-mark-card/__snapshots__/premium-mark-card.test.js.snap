// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PremiumMarkCard component snapshot Should correctly render PremiumMarkCard component 1`] = `
<div
  className="place-card__mark"
>
  <span>
    Premium
  </span>
</div>
`;
